src/CMakeLists.txt

```diff
@@ -14,6 +14,22 @@ add_custom_command(
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/gen_types.py)
 
+add_custom_command(
```

Comment:
This is intended for developers that will modify the .h files directly.  If you just build from the source, it shouldn't have to regenerate since the files are unmodified.